src/reference/config/writing-platform-configs.rst

```diff
@@ -49,277 +49,276 @@ Why Were Platforms Introduced?
-same file system. It defaults to the name of the platform.
+same file system. Cylc relies on the site setup file ``global.cylc`` to determine
+which platforms share install targets. Cylc will then use this information to
+make the correct installations on remote platforms, including installation of
+files, creation of :cylc:conf:`global.cylc[install][symlink dirs]` and
+authentication keys to enable secure communication between platforms.
 
-For example, your Cylc scheduler hosts might share a file system with a
-compute cluster. Cylc does not need to install files on this cluster. The
-cluster and scheduler hosts are different platforms, but share an install
-target.
+Note, if missing from configuration, the install target will default to the
+platform name. If incorrectly configured, this will cause errors in
+:ref:`RemoteInit`.
 
-But you might also have mirrored clusters, each with their own file system.
-Each cluster would be a platform with its own install target.
+If you log into one system and see the same files as on another, then these two
+platforms will require the same install target in ``global.cylc`` config file.
 
+Example Platform Configurations
+-------------------------------
 
-Example Platforms
------------------
+Detailed below are some examples of common platform configurations.
 
-On the Scheduler Host (Cylc Server)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Submit PBS Jobs from Localhost
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- **There is a built in localhost platform**
+- **The** :term:`scheduler` **runs on the** ``localhost`` **platform.**
+- **Platforms can share hosts without sharing job runners.**
 
 .. admonition:: Scenario
 
-   You want to allow users to submit small jobs to the scheduler host:
-
-If a job doesn't set a platform it will run on the Cylc scheduler host
-using a default ``localhost`` platform.
-
-Simple Remote Platform
-^^^^^^^^^^^^^^^^^^^^^^
-
-- **Platforms don't need to be complicated**
-- ``install target`` **specifies a file system for the task using that platform**
-
-.. admonition:: Scenario
+   You have a cluster where you can submit jobs from the Cylc scheduler host
+   using PBS.
 
-   Users want to run background jobs on a single server,
-   which doesn't share a file system with the workflow host.
+The ``localhost`` platform is the Cylc Scheduler host, as configured in
+:cylc:conf:`global.cylc[scheduler][run hosts]available`. This is the host that
+the workflow will start on.
+Our platform ``pbs_cluster`` shares this ``localhost`` host and setting the
+install target to ``localhost`` ensures that Cylc knows this platform does not
+require remote initialization.
 
 .. code-block:: cylc
    :caption: part of a ``global.cylc`` config file
 
    [platforms]
-       [[myhost]]
-           hosts = myhost
-           install target = myhost
+       [[localhost]]
+           hosts = localhost
+           install target = localhost
```

Comment:
This isn't required. I assume you want to include it for clarity?
You could have it commented out? e.g.
```
       # The localhost platform is available by default
       #[[localhost]]
       #    hosts = localhost
       #    install target = localhost
```
Or just remove it and mention it in the paragraph above.